code/procs/gamehelpers.dm

```diff
@@ -247,6 +249,13 @@ var/obj/item/dummy/click_dummy = new
 
 	return viewers(Depth, Center) + get_viewing_AIs(Center, 7)
 
+/**
+* Determines whether or not the user can remote access devices.
+* This is typically limited to Borgs and AI things
+*/
+/proc/can_access_remotely(mob/user)
```

Comment:
I would consider putting this proc on `/obj` (possibly even `/obj/machinery`), then access via `src.can_access_remotely(user)`. This would allow for overriding for specific machinery if needed, or based on some conditions of the machinery in addition to the user.

Reply:
Prototype added to object for ease of overriding by non-machinery where applicable (returns FALSE).  Implementation of set of mobs capable of remote access also moved to `obj` as `can_access_remotely_default`.   This allows any object to override with a consistent behavior.  Updated override in `/obj/machinery` to factor in status bit-field.